awesomemotive/affiliatewp-leaderboard
# issue/29 - Bump WordPress minimum requirements

Issue #29


**Housekeeping**


**Proposed changes**


## How to test it
1. Install and activate AffiliateWP Leaderboard add-on
2. Confirm the plugin works as expected without any PHP errors.
3. Create a new page with the shortcode `[affiliate_leaderboard]` and confirm the page shows the leaderboard
